extension/data.ts

```diff
@@ -613,32 +616,19 @@ class RcxDict {
         'K',  'Gakken Kanji Dictionary Index',
         'W',  'Korean Reading',
     */
```

Comment:
I'm not sure if we want to keep this comment up here.

Reply:
I guess this comment is meant to describe kanji info which is present in the dictionary but that we don't extract for rikaikun.

I'm not sure why we don't just include all of them (since they're optional) but having the comment here seems like a quick way to answer questions about missing indicies. (that said, this is probably out of date so maybe linking to the [canonical source of truth](http://www.edrdg.org/wiki/index.php/KANJIDIC_Project) would also be good)

That raises some additional questions to me though:
1. Should we periodically look at the fields available and add them to rikaikun (it is pretty easy to whitelist a new info field iirc)

2. Is replacing the comment with just a link to the canonical source enough. One thing this comment does is make it easy to tell which fields are intentionally not included but maybe it would be better to strive to include all fields and periodically check for new fields (I'm guessing update-db could be made to print a message when such an update is available)

tl;dr
Probably leave for now maybe adding the link to the documentation; we can think about the deeper questions later.

Reply:
I added a small explanation and the link you mentioned to the comment. Thanks!